### ISSUE TITLE
Cannot run example on ubuntu 17.10 nor debian 8/9

### ISSUE THREAD

**Issue by vaclav-kasal** [peachpiecompiler/peachpie#120]
Hello,

I can't get work example with debian 8/9 nor ubuntu 17.10
I've done steps from https://www.microsoft.com/net/download/linux and from https://www.peachpie.io/getstarted?

Last step `dotnet run` returns error with following output:
```bashvagrant@jessie:~$ dotnet run
/home/vagrant/.nuget/packages/peachpie.net.sdk/0.7.0-ci00318/build/Peachpie.NET.Core.Sdk.targets(148,5): error MSB3073: The command "dotnet compile-php @obj/Debug/netcoreapp1.0/compile-php-args.rsp" exited with code 131. [/home/vagrant/vagrant.msbuildproj]

The build failed. Please fix the build errors and run again.
```
Steps to reproduce - `mkdir stretch; cd stretch; vagrant init debian/stretch64; vagrant ssh` and follow steps from https://www.microsoft.com/net/learn/get-started/linuxdebian https://www.peachpie.io/getstarted? 

Which linux distro is working? Thanks.

**Comment by jakubmisek** [comment]
Hi,

The `templates` were outdated. Version `0.7.0` did have probably some issues. Try `Install Peachpie .NET Templates` again if it helps and create a new sample project. Your project should reference version `0.8.0-*`

**Comment by jakubmisek** [comment]
@vaclav-kasal did it help?

**Comment by jakubmisek** [comment]
fixed with `0.8.0`, currently samples still run on .NET Core SDK 1.1

**Comment by jakubmisek** [comment]
latest templates `0.8.8` target .net core 2.0 and fix ubuntu build
